Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -999,14 +999,15 @@ function Set-AvsDrsElevationRule {
 
 <#
     .Synopsis
-     Gets all the storage policies available to set on a VM 
+     Gets all the storage policies available to set on a VM.
 #>
 function Get-StoragePolicies {
     [AVSAttribute(3, UpdatesSDDC = $False)]
     
     $StoragePolicies
     try {
-        $StoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop
+        $JetstreamStoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop | Where-Object {$_.CommonRule -match '^jetdr.*'} | Select-Object Name
+        $StoragePolicies = Get-SpbmStoragePolicy -ErrorAction Stop | Select-Object Name, AnyOfRuleSets | Where-Object {$JetstreamStoragePolicies.Name -notcontains $_.Name}
```

Comment:
Suggest filtering by the vSAN namespace here via the Namespace parameter to exclude non-VSAN storage policies such as those that may be provided by 3rd parties.